buildSrc/src/main/kotlin/com/orange/ods/gradle/Versions.kt

```diff
@@ -23,6 +23,7 @@ object Versions {
     const val browser = "1.3.0"
     const val compose = "1.2.0-rc02"
     const val core = "1.7.0"
+    const val customViewPoolingContainer = "1.0.0-rc01"
```

Comment:
Version 1.0.0
July 27, 2022
androidx.customview:customview-poolingcontainer:1.0.0 is released.